articles/install-guide/setup/index.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Setup Overview
+author: gillenhaalb
+ms.author: gillenhaalb@gmail.com
+ms.date: 7/19/2019
+ms.topic: article
+uid: microsoft.quantum.install.setup
+---
+
+# Setup Overview #
+
+To begin writing and running quantum programs on your own computer, follow these instructions for installing the Quantum Development Kit.
+Whether on Windows, Linux, or macOS, you will need the right machinery in the background, so the first step below is to download the [.NET Core SDK](https://dotnet.microsoft.com/).
```

Comment:
I'm not sure this line is necessary, since the instructions directly follow. Is the intent to state that the QDK is supported across all 3 OS's?

Reply:
At least part of my thinking with suggesting this paragraph was that the reference rendering looked really awkward with a `#` header immediately followed by a `##` header. My hope was that some slight text helps space things up a bit and flow a bit better.